lib/fluent/plugin/in_tail.rb

```diff
@@ -65,6 +65,8 @@ def initialize
     config_param :read_lines_limit, :integer, default: 1000
     desc 'The interval of flushing the buffer for multiline format'
     config_param :multiline_flush_interval, :time, default: nil
+    desc 'Enable the option to capture unmatched lines.'
+    config_param :enable_catch_all, :bool, default: false
```

Comment:
`emit_unmatched_lines` or `emit_parse_failed_lines` is more better.
`enable_catch_all` is unclear name for me.